src/rules/requireReadonlyReactProps.js

```diff
@@ -1,6 +1,7 @@
 const schema = [];
 
 const reComponentName = /^(Pure)?Component$/;
+const reReadOnly = /^\$(ReadOnly|FlowFixMe)$/;
```

Comment:
`$FlowFixMe` was just an example. It can be anything defined in `suppress_type`.

Reply:
how would we know what's defined there (from within a plugin rule)?

Reply:
You don't. This is why I was suggesting to not throw an error in cases where you don't know it's wrong for sure. :)

Reply:
Maybe allow to specify the suppress type name in an option of the rule?